sodapy/__init__.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, domain, app_token, username=None, password=None,
                                session_adapter["adapter"])
             self.uri_prefix = session_adapter["prefix"]
         else:
-            self.uri_prefix = "https"
+            self.uri_prefix = "https://"
```

Comment:
`session.mount()` works when passed `mock` but not when `https` - for https to work, it needs to be passed `https://`. I'm not sure why this is, but the [documentation](http://docs.python-requests.org/en/latest/user/advanced/#transport-adapters) suggests this approach fits fine. It required a few changes elsewhere, removing `://` from a few string constructors. The alternative to doing that is to inject a `://` into the `session.mount()` call, but I chose not to do that in case, in theory, you wanted to use a longer prefix like `http://google.com` (which would work with this approach)